### PR TITLE
Update ilp july

### PR DESCRIPTION
**Pull request by jmealing**

### CODE REVIEW

File: mise-en-page/accueil-institution-directives.html [hunk 1]
```diff
@@ -153,7 +150,7 @@ <h1 property="name" id="wb-cont">[Nom de l'institution]</h1>
 					<div class="row guidance-details">
 						<div class="col-md-8">
 							<details>
-								<summary><h2 class="h4">Directives&nbsp;:</h2> Titre, image et bouton vers une super-tâche <i class="fas fa-info-circle"></i></summary>
```
Comment (delisma) [inline mise-en-page/accueil-institution-directives.html:156]: Didn't we want to remove the section in the guidance pieces instead?

File: mise-en-page/accueil-institution-directives.html [hunk 2]
```diff
@@ -167,7 +164,8 @@ <h3>Image</h3>
 									<p>La zone de 1200x726 derrière le H1 peut être utilisée pour une bannière ou une image. L'image n'a pas besoin d'occuper tout l'espace et peut être personnalisée au besoin.</p>
 									<p>L'image et le H1 doivent être visuellement distincts de la <a href="https://www.canada.ca/">page d'accueil de Canada.ca</a> afin d'éviter toute confusion entre la page d'accueil de l'institution et celle de Canada.ca.</p>
 									<p>Si vous n'utilisez pas d'image, l'arrière-plan devrait être blanc.</p>
-									<p><b>Variation&nbsp;:</b> Appliquez la classe <code>ilp-cover-hide-xxs</code> ou la classe <code>ilp-cover-hide-xs</code> en plus de <code>ilp-cover-img</code> pour cacher l'image pour les appareils mobiles.</p>
+									<p><b>Variation&nbsp;:</b> The class <code>bg-hide-sm</code> hides the image on tablet, <code>bg-hide-xs</code> hides the image on mobile and
+									<code>bg-right</code> positions the image on the right.</p>
```
Comment (delisma) [inline mise-en-page/accueil-institution-directives.html:168]: Will this be translated in French later? Do you need some help for that?

Comment (jmealing) [reply mise-en-page/accueil-institution-directives.html:168]: Yes it's being previewed by Emilie and won't be merged until it's all ready

File: mise-en-page/accueil-institution-directives.html (hunk 1 above)
Comment (jmealing) [reply mise-en-page/accueil-institution-directives.html:156]: A bunch of guidance pieces have h3s in them, which like... technically are headings? 

I can take out the `<sections>`

**Comment by jmealing** [comment]
This is meant to be a draft, I can't seem to change it... Cade bumped my arm

**Comment by jmealing** [comment]
Found draft!

File: mise-en-page/accueil-institution-directives.html (hunk 1 above)
Comment (delisma) [reply mise-en-page/accueil-institution-directives.html:156]: I was thinking more about removing the `h3`s to replaced them with `p.h3` and removing the sections. That way the guidances are not part of the document outline. Unless we want the guidances to be included than removing the sections should be okay.

**Pull request merged**